apps/web/src/content/blog/en/angular-mobile-app-capacitor.md

```
@@ -237,86 +237,204 @@ npx cap sync
+  icons: true,
+  colors: { dynamic: true },
+  tabs: [
+    { id: 'home', title: 'Home', icon: { svg: '...' } },
+    { id: 'settings', title: 'Settings', icon: { svg: '...' } },
+  ],
+});
+
+await NativeNavigation.addListener('tabSelect', ({ id }) => {
+  router.navigate([`/${id}`]);
+});
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)